src/alertsmanagement/azext_alertsmanagement/tests/latest/test_alertsmanagement_scenario.py

```diff
@@ -0,0 +1,81 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+import unittest
+
+from azure_devtools.scenario_tests import AllowLargeResponse
+from azure.cli.testsdk import (ScenarioTest, ResourceGroupPreparer)
+
+
+TEST_DIR = os.path.abspath(os.path.join(os.path.abspath(__file__), '..'))
+
+
+class AlertsScenarioTest(ScenarioTest):
+
+    @ResourceGroupPreparer(name_prefix='cli_test_alertsmanagement_action_rule_')
+    def test_alertsmanagement_action_rule(self, resource_group):
+        subs_id = self.get_subscription_id()
+        rg_id = '/subscriptions/{}/resourceGroups/{}'.format(subs_id, resource_group)
+        self.kwargs.update({
+            'rg_id': rg_id
+        })
+        self.cmd('az alertsmanagement action-rule create '
+                 '--resource-group {rg} '
+                 '--name rule1 '
+                 '--location Global '
+                 '--status Enabled '
+                 '--rule-type Suppression '
+                 '--scope-type ResourceGroup '
+                 '--scope {rg_id} '
```

Comment:
When --scope-type is Resource, scope will take a resource id? What kind of resources are supported?

Reply:
Yes. Any resource that may generate alerts.